test/Mono.Linker.Tests.Cases/RequiresCapability/Dependencies/RequiresAttributeInCopyAssembly.cs

```diff
@@ -9,48 +9,56 @@
 
 namespace Mono.Linker.Tests.Cases.RequiresCapability.Dependencies
 {
-	public class RequiresUnreferencedCodeInCopyAssembly
+	public class RequiresAttributeInCopyAssembly
 	{
-		public RequiresUnreferencedCodeInCopyAssembly ()
+		public RequiresAttributeInCopyAssembly ()
 		{
 		}
 
 		[RequiresUnreferencedCode ("Message for --Method--")]
+		[RequiresAssemblyFiles (Message = "Message for --Method--")]
```

Comment:
The fact that the two attributes can't look the same is really not ideal. I filed https://github.com/dotnet/runtime/issues/55274 to track changing it.